configuring.tex

```diff
@@ -1,424 +1,185 @@
-page is located. Click the \textbf{Download} link to retrieve the latest \textbf{SD essentialsNoROM.rar} file.
+{\bf Swap stereo channels} switches the stereo mix to use the opposite speakers.
 
-Note that while this fileset does not contain a ROM, there are future plans for it to include the freely available ROM from the Open ROMs project.
+{\bf Audio amplifier} controls the built-in amplifier on the 3.5mm audio jack. Set this to {\bf on} when using headphones or another device that expects an amplified signal. Set this to {\bf off} for a line-level signal.
 
-\includegraphics[width=\linewidth]{images/latest_support_files.png}
-\fi
+\subsection{Network}
+\index{Configuration!Network}
 
-\section{On-boarding}
-\label{onboarding}
+The {\bf Network} page gives you the opportunity to adjust the MAC address of the Ethernet port. The MEGA65 does not have a hardware-assigned MAC address. Instead, it uses the value entered here.
 
-On first launch of your MEGA65, you will see the on-boarding screen:
+If the MAC address is set to all zeros, press the \megakey{R} key to generate a random address. Networking features will not function with an MAC address set to all zeros.\index{Configuration!MAC address}\index{Networking!MAC address}
 
 \begin{center}
-  \includegraphics[trim= 10mm 15mm 10mm 10mm,clip,width=0.7\linewidth]{images/img011_final_boot_01.png}
+  \includegraphics[width=0.7\linewidth]{images/ss-m65config-5.png}
 \end{center}
 
-Here you can select and test you screen configuration.
+\subsection{Done}
 
-For example, press \specialkey{TAB} to switch to PAL 50HZ:
-\index{Display!Setting PAL/NTSC}
-\begin{center}
-  \includegraphics[width=0.7\linewidth]{images/img011_final_boot_02.png}
-\end{center}
-
-Then press \megakey{SPACE}, followed by \megakey{Y} to test the new video mode:
+The {\bf Done} page lets you exit the Configuration Utility. If you have made changes that you want to keep, select {\bf Save as defaults and exit.} You can also abandon changes, restore the factory default settings, or completely restart to the on-boarding screen.
 
 \begin{center}
-  \includegraphics[trim= 15mm 10mm 10mm 15mm,clip,width=0.7\linewidth]{images/img011_final_boot_03.png}
+  \includegraphics[width=0.7\linewidth]{images/ss-m65config-save.png}
 \end{center}
 
-Press \megakey{K} to keep the new video mode:
 
-\begin{center}
-  \includegraphics[trim= 20mm 20mm 10mm 25mm,clip,width=0.7\linewidth]{images/img011_final_boot_04.png}
-\end{center}
+\newpage
+\section{Introducing SD Cards}
+\label{sec:introducing-sd-cards}
+\index{SD Cards}
 
-Press \specialkey{RETURN} to complete the configuration:
+Your MEGA65 is equipped with two SD card slots: a full-size SD card slot inside the case, and a microSD card slot accessible from the rear of the computer.\index{SD Cards!Locations} The MEGA65 uses the SD card for storing configruation settings, loading the operating system, updating the firmware, and storing your software and data as virtual disk images.
```

Comment:
To be honest, I removed these originally because I don't believe the trap door is actually a good way to access the internal SD card. The design had good intentions but the positions didn't line up in the final product and it's quite difficult to remove the SD card this way.

Based on your suggestion I tried messing with the trap door again and decided I could go either way, so I have added them back.